101-bundle-json.md

```diff
@@ -764,6 +767,7 @@ The above declares three actions: `io.cnab.status`, `io.cnab.migrate` and `io.cn
 
 Each action is accompanied by a description, which contains the following fields:
 
+- `display`: The human-readable name of the operation. If omitted, tools MAY use the value of the `actions` key (e.g. `io.cnab.dry-run`). User agents that display the display name SHOULD disambiguate if two or more actions have the same `display` value.
```

Comment:
nit: We use `title` in the `definitions` specs, which comes from JSONSchema, see https://github.com/cnabio/cnab-spec/blob/36d7b4817de6902142fc338549c8d93aa94b37a5/101-bundle-json.md#L411